rule_match.go

```diff
@@ -85,56 +92,71 @@ func (mr MatchedRule) details() string {
 	return log.String()
 }
 
-func (mr MatchedRule) matchData() string {
-	v := mr.MatchedData.Variable.Name()
-	if mr.MatchedData.Key != "" {
-		v += fmt.Sprintf(":%s", mr.MatchedData.Key)
-	}
-	value := mr.MatchedData.Value
-	if len(value) > 200 {
-		value = value[:200]
-	}
+func (mr MatchedRule) matchData(matchData MatchData) string {
 	log := &strings.Builder{}
-	if mr.Rule.operator != nil {
-		log.WriteString(fmt.Sprintf("Matched \"Operator %s matched %s at %s.",
-			"", value, v))
-	} else {
-		log.WriteString("Matched.\"")
+	for _, matchData := range mr.MatchedDatas {
+		v := matchData.Variable.Name()
+		if matchData.Key != "" {
+			v += fmt.Sprintf(":%s", matchData.Key)
+		}
+		value := matchData.Value
+		if len(value) > 200 {
```

Comment:
While I see that this was here before, can we switch this 200 to a package const?

Maybe downstream someone wants to log more than that, so it would be awesome if it can be configured...

Reply:
Will make such change in another PR. There are various places this 200 limit is used.